cassandra/skilldirectoryschema.cql

```diff
@@ -40,6 +40,12 @@ CREATE TABLE IF NOT EXISTS skill_directory_keyspace.skillreviews (
     PRIMARY KEY (skill_id, id)
 );
 
+CREATE TABLE IF NOT EXISTS skill_directory_keyspace.skillicons (
+    id       uuid, // Will contain associated skill's id
```

Comment:
Any reason you chose `id` over `skill_id`?  Because this isn't a unique id, and more of a reference to a skill's id, I wonder if this makes sense